src/Traversals/Traversals.jl

```diff
@@ -11,7 +11,6 @@ import Base:show
 
 """
     TraversalError <: Exception
-
```

Comment:
you shouldn't be removing these blank lines. The docstrings have a particular format.